index.html

```diff
@@ -14,20 +14,172 @@
   <header class="header">
     <nav class="navbar">
       <a class="logo navbar-item position"><img src="images/image_geometry_1.svg" alt="A geometry image"/></a>
-      <i class="fas fa-bars navbar-item icon position"></i>
+      <i class="fas fa-bars navbar-item icon position primary-color"></i>
     </nav>
   </header>
   <section class="summary">
-    <div class="name position left16 align-left no-margin">
+    <div class="name position left16 align-left no-margin primary-color">
       <h1 class="no-margin">Ricky
       Mormor</h1>
     </div>
-    <div class="intro position left16 align-left width288">
+    <div class="intro position left16 align-left width288 poppins">
       <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
       </div>
 
-      <button class="cta position typography width288">Start collaboration</button>
+      <button class="cta position typography width288 primary-color">Start collaboration</button>
+  </section>
+
+  <section class="projects-section no-margin">
+    <div>
+      <img src="images/image-geometry_projects.png" class="head-image position" alt="A nice geometry image">
+    </div>
+    <div class="primary-color projects-head position typography left16">
+      <h2>Projects</h2>
+    </div>
+
+    <div class="projects-list no-margin">
+      <div class="project-cards no-margin">
+        <div class="cards">
+          <div class="card1 fix-card"></div>
+          <div class="cards-detail dark-bg">
+            <p class="poppins">name goes here</p>
+            <button>HTML/CSS</button>
+            <button>Ruby on Rails</button>
+            <button> Javascript</button>
+            <div>
+              <button class="go-to dark-bg">See this project
+                <i class="fas fa-arrow-right"></i>
+              </button>
```

Comment:
- [ ] These elements are buttons and hence making them links. Please add a hover effect over them so that a user can recognize them as links. Don't worry about any JavaScript functionality.

Reply:
I did make them links but I had linter errors saying " Error  The element “button” must not appear as a descendant of the “a” element." So I gave them a type button attribute instead. Does that work?